### PR TITLE
[Brand] Added MeshMasters brand kit

### LINKED ISSUE #2316: [Brand] Add MeshMasters brand kit
**Current Behavior**
https://layer5.io/company/brand does not have a section for MeshMasters.

**Desired Behavior**
Create a new section for MeshMasters.

**Kit** 
[meshmasters-brand-kit.zip](https://github.com/layer5io/layer5/files/7449528/meshmasters-brand-kit.zip)

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by vedant-kakde**
Signed-off-by: Vedant Kakde <69970950+vedant-kakde@users.noreply.github.com>

**Description**

This PR fixes #2316 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 22bbcaa6ce73bc39c49700c5055885bf7ee9178f at: https://618ca214640969a995bce5cd--layer5ng.netlify.app

**Comment by Mr-DG-Wick** [comment]
Hi @vedant-kakde. Can the png and the svg files be shifted to different folders under  `src/assets/images/` like the way it is done in the SMP brand kit.
Screenshot (1090)

**Comment by vedant-kakde** [comment]
@Mr-DG-Wick I think it is not required as the brand kit will be downloaded in complete .zip file.

**Comment by Mr-DG-Wick** [comment]
ohh alright @vedant-kakde

**Review by Mr-DG-Wick — COMMENTED** [review]
LGTM!

File: src/sections/Company/Brand/Brand-components/meshmaster.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      </Row>
+      <Row>
+        <Col xs={12}>
+          <h2 className="layerH3 in">
+            Colors
+          </h2>
+        </Col>
+        <Col>
+          <p>
+            The Meshmaster color palette consists of the primary Image Hub
```
Comment (warunicorn19) [inline src/sections/Company/Brand/Brand-components/meshmaster.js:67]:
```suggestion
            The MeshMaster color palette consists of the primary Image Hub
```

File: src/sections/Company/Brand/Brand-components/meshmaster.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <Col xs={12} sm={6} className="download-button">
+          <a href="/brand/meshmasters-brand-kit.zip">
+            <Button primary title="Download Logo Assets" external={true}>
+              <FiDownloadCloud size={21} className="icon-left" />
+            </Button>
+          </a>
+        </Col>
+        <Col>
+          <p>
+            The Meshmaster mark includes the Meshmaster name & logo, and any
```
Comment (warunicorn19) [inline src/sections/Company/Brand/Brand-components/meshmaster.js:29]:
```suggestion
            The MeshMaster mark includes the Meshmaster name & logo, and any
```

File: src/sections/Company/Brand/Brand-components/meshmaster.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import MeshMasterIcon from   "../../../../assets/images/meshmaster/meshmaster-icon.png";
+import MeshMasterText from   "../../../../assets/images/meshmaster/meshmaster-text.png";
+
+const MeshMasterBrand = () => {
+  return (
+    <div className="post-content">
+      <Row className="brand-section">
+        <Col xs={12} sm={6}>
+          <h1 className="layerH3">
+          Meshmaster
```
Comment (warunicorn19) [inline src/sections/Company/Brand/Brand-components/meshmaster.js:17]:
```suggestion
          MeshMaster
```

File: src/sections/Company/Brand/index.js [hunk 4]
```diff
@@ -272,6 +278,16 @@ const Brand = () => {
...
             </p>
+            <p
+              className={`header_link ${
+                visibleSection === "MeshMaster" ? "selected" : ""
+              }`}
+              onClick={() => {
+                scrollTo(meshMasterRef.current);
+              }}
+            >
+              <span> Meshmasters </span>
```
Comment (warunicorn19) [inline src/sections/Company/Brand/index.js:289]:
```suggestion
              <span> MeshMasters </span>
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Review by warunicorn19 — APPROVED** [review]

**Comment by vedant-kakde** [comment]
Thanks @warunicorn19 😄

**Comment by warunicorn19** [comment]
no problem😃

**Pull request merged**